src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex.cs

```diff
@@ -58,24 +62,31 @@ public static async Task PrecalculateAsync(Document document, CancellationToken
 
         public static async Task<SyntaxTreeIndex> GetIndexAsync(
             Document document,
+            bool loadOnly,
             CancellationToken cancellationToken)
```

Comment:
see above comment about  having another entrypoint that doesn't have this parameter.  then revert all teh locations that you updated to pass `loadOnly: false`.